src/widgets/splits/SplitInput.hpp

```diff
@@ -188,6 +188,9 @@ class SplitInput : public BaseWidget
 
     QPropertyAnimation backgroundColorAnimation;
 
+private:
```

Comment:
Hmm, this `private` should be a bit higher (after the last `override`). Looks like this was never added before and everything is protected now. Would be fine if you leave it out too.